.github/workflows/test-lts-container-images.yaml

```diff
@@ -0,0 +1,50 @@
+name: Test LTS Container Images
+
+# TODO: When to Trigger?
+on: [push]
```

Comment:
Looking at the available options, I think this makes the most sense.
https://docs.github.com/en/actions/reference/events-that-trigger-workflows

Reply:
:+1:  I might add pull_request